plugins/modules/ntnx_lcm_config_info_v2.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ntnx_lcm_config_info_v2
+short_description: Fetch LCM Configurations
+description:
+    - This module fetches LCM configurations.
+    - Fetch LCM configurations using cluster external ID.
+version_added: 2.0.0
```

Comment:
```suggestion
version_added: 2.1.0
```
Please update everywhere